src/components/content/review/ApproveOrRejectServiceTemplate.tsx

```diff
@@ -28,8 +28,14 @@ export const ApproveOrRejectServiceTemplate = ({
 }): React.JSX.Element => {
     const { TextArea } = Input;
     const [comments, setComments] = useState<string>('');
+    const [error, setError] = useState<string>('');
     const handleOk = () => {
         if (isApproved !== undefined) {
+            if (comments.trim() === '') {
+                setError('Comments are required.');
+                return;
+            }
+
             const request: ApproveOrRejectRequestParams = {
```

Comment:
The field 'reviewComment' in model `ReviewServiceTemplateRequest` must be notNull and notEmpty. But I don't see any change in the code of backend.

![image](https://github.com/user-attachments/assets/4615249b-9002-4e72-a13b-4cd102eb8d8b)

Reply:
backend updated:
![image](https://github.com/user-attachments/assets/e5461c07-90f0-475c-a4da-92c3562c70a1)